clippy_dev/src/main.rs

```diff
@@ -180,7 +181,8 @@ fn get_clap_config() -> ArgMatches {
                         .short('n')
                         .long("name")
                         .help("Name of the new lint in snake case, ex: fn_too_long")
-                        .required(true),
+                        .required(true)
+                        .value_parser(|name: &str| Ok::<_, Infallible>(name.replace("-", "_"))),
```

Comment:
```suggestion
                        .value_parser(|name: &str| Ok::<_, Infallible>(name.replace('-', "_"))),
```

Clippy is not happy with itself.